apps/rush-lib/src/cli/actions/ListAction.ts

```diff
@@ -1,76 +1,98 @@
-import * as Table from 'cli-table';
+import { BaseRushAction } from "./BaseRushAction";
+import { RushCommandLineParser } from "../RushCommandLineParser";
+import { CommandLineFlagParameter } from "@microsoft/ts-command-line";
+import { RushConfigurationProject } from "../../api/RushConfigurationProject";
+import * as Table from "cli-table";
 
 export class ListAction extends BaseRushAction {
   private _version: CommandLineFlagParameter;
   private _path: CommandLineFlagParameter;
   private _fullPath: CommandLineFlagParameter;
+  private _jsonFlag: CommandLineFlagParameter;
 
   public constructor(parser: RushCommandLineParser) {
     super({
-      actionName: 'list',
-      summary: 'List package information for all projects in the repo',
+      actionName: "list",
+      summary: "List package information for all projects in the repo",
       documentation:
-        'List package names, and optionally version (--version) and ' +
-        'path (--path) or full path (--full-path), for projects in the ' +
-        'current rush config.',
+        "List package names, and optionally version (--version) and " +
+        "path (--path) or full path (--full-path), for projects in the " +
+        "current rush config.",
       parser
     });
   }
 
   protected onDefineParameters(): void {
     this._version = this.defineFlagParameter({
-      parameterLongName: '--version',
-      parameterShortName: '-v',
-      description: 'If this flag is specified, the project version will be ' +
-        'displayed in a column along with the package name.'
+      parameterLongName: "--version",
+      parameterShortName: "-v",
+      description:
+        "If this flag is specified, the project version will be " +
+        "displayed in a column along with the package name."
     });
 
     this._path = this.defineFlagParameter({
-      parameterLongName: '--path',
-      parameterShortName: '-p',
-      description: 'If this flag is specified, the project path will be ' +
-        'displayed in a column along with the package name.'
+      parameterLongName: "--path",
+      parameterShortName: "-p",
+      description:
+        "If this flag is specified, the project path will be " +
+        "displayed in a column along with the package name."
     });
 
     this._fullPath = this.defineFlagParameter({
-      parameterLongName: '--full-path',
-      parameterShortName: '-f',
-      description: 'If this flag is specified, the project full path will ' +
-        'be displayed in a column along with the package name.'
+      parameterLongName: "--full-path",
+      parameterShortName: "-f",
+      description:
+        "If this flag is specified, the project full path will " +
+        "be displayed in a column along with the package name."
+    });
+
+    this._jsonFlag = this.defineFlagParameter({
+      parameterLongName: "--json",
+      description: "If specified,  a json file of result will be generated."
```

Comment:
I think it would be best if we could match the sentence structure of the other flags for uniformity's sake. Something like:

"If this flag is specified, the result will be generated in JSON format."

Reply:
```suggestion
      description: "If specified, output will be in JSON format."
```